Italian/guardare/guarda.md

```diff
@@ -0,0 +1 @@
+mi guardo intorno e cerco di capire cosa accade. Vedo le pareti che si restringono, dei simpatici, carini e piccoli elfi che ballano e cantano e..comprendo che i funghi "speciali" che ho mangiato ieri in realta' erano allucinogeni!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.